src/App.js

```diff
@@ -4,9 +4,18 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    word: '',
```

Comment:
```suggestion
    query: '',
```
`word` is too common